.github/workflows/test.yml

```diff
@@ -54,3 +54,6 @@ jobs:
       with:
         name: coverage.xml
         path: ./coverage.xml
+
+    - name: Run Linting
+      run: make lint
```

Comment:
- `make lint` runs `flake8` and `isort`.
- `make format` runs `isort` and `black`.
- Shouldn't `make lint` also run `black --check`?
- Do not use `black --check --diff`, as that does not use cache.